### PR TITLE
General code of conduct page

### PR DESCRIPTION
**Pull request by rantahar**
Missing members of the committee. Could this be the board?

Review: 
 - This is written to apply to both event participants and to association members. Does it sound clunky?
 - Typos, mistakes...

### CODE REVIEW

File: content/about/code-of-conduct.md [hunk 1]
```diff
@@ -2,4 +2,105 @@
...
+Be kind to others. Do not insult or put down others.
+
+Behave professionally. Remember that harassment, unprofessional remarks and
+messages, and exclusionary jokes are not appropriate in Nordic-RSE.
+
+Members violating these rules may have their membership revoked by the board.
+Participants to events violating these rules may be asked to leave without a
+refund at the sole discretion of the conference organisers.
+
+Thank you for helping make this a welcoming, friendly event for all.
```
Comment (annefou) [inline content/about/code-of-conduct.md:44]: Maybe I would not leave this sentence here _Thank you for helping make this a welcoming, friendly event for all._

File: content/about/code-of-conduct.md [hunk 2]
```diff
@@ -2,4 +2,105 @@
...
+Excessive swearing is not appropriate.
+
+If a member engages in behaviour that violates this code of conduct, the board
+of the association may take any action they deem appropriate, including
+warning the offender or may revoking their membership without refund of the
+membership fee.
+
+If a participant at a Nordic-RSE event violates this code of conduct,
+organisers may take any action they deem appropriate, including
+warning the offender or expulsion from the workshop with no refund.
```
Comment (annefou) [inline content/about/code-of-conduct.md:69]: Should we also specifically cover the case where a board member is involved?
H/she is a member of the association so we are covered but this person should also be excluded from the board (and should not be involved in the exclusion decision).

File: content/about/code-of-conduct.md [hunk 3]
```diff
@@ -2,4 +2,105 @@
...
+
+
+### Procedure for reporting harassment
+
+If you are being harassed, notice that someone else is being harassed, or have
+any other concerns, please contact any member of the code of conduct committee:
```
Comment (annefou) [inline content/about/code-of-conduct.md:75]: above, we mentioned that the board of the association can take action. Will the COC committee provide recommendation to the board of the association?

**Comment by rantahar** [comment]
Fixes issue #284

File: content/about/code-of-conduct.md (hunk 1 above)
Comment (rantahar) [reply content/about/code-of-conduct.md:44]: Missed that one :) Will fix.

File: content/about/code-of-conduct.md (hunk 2 above)
Comment (rantahar) [reply content/about/code-of-conduct.md:69]: I added a chapter about this. What do you think?

File: content/about/code-of-conduct.md (hunk 3 above)
Comment (rantahar) [reply content/about/code-of-conduct.md:79]: I wasn't sure if the committee should be the same as the board. Maybe separate, but they make recommendations to the board as you suggest.

Comment (annefou) [reply content/about/code-of-conduct.md:79]: Yes. it sounds right. Thanks,

**Review by annefou — APPROVED** [review]

**Comment by annefou** [comment]
@bast shall we merge this PR?

**Comment by bast** [comment]
Yes. Sorry for not commenting earlier. Let's merge and if we need to adjust, we can iterate from the present document. Thanks a lot for drafting and improving!

**Pull request merged**